bin/workflows/pycbc_make_offline_search_workflow

```diff
@@ -187,8 +188,76 @@ insps = wf.merge_single_detector_hdf_files(workflow, hdfbank,
                                            insps, output_dir,
                                            tags=['full_data'])
 
+# Check for the only-do section. If inspiral=true then it will only do the workflow up until the merged inspiral hdf files
+if 'only-do' in workflow.cp.sections():
+    only_do_inspiral = workflow.cp.get('only-do', 'inspiral', fallback='false').lower() == 'true'
+else:
+    only_do_inspiral = False
+
+if only_do_inspiral:
```

Comment:
This looks like we could combine with the code at the end to remove the redundance here. Maybe put into a function and then we can simply call from either location?